torch/onnx/utils.py

```diff
@@ -115,6 +115,19 @@ def export(model, args, f, export_params=True, verbose=False, training=False,
         strip_doc_string (bool, default True): if True, strips the field
             "doc_string" from the exported model, which information about the stack
             trace.
+        example_outputs: example outputs of the model that is being exported.
+        dynamic_axes (dict<string, dict<int, string>> or dict<string, list(int)>, default empty dict): 
+            a dictionary to specify dynamic axes of input/output. The dictionary takes an input/output 
+            name as key and maps it one of the values below:
+            (1)A list of integers specifiying the dynamic axes of provided input. In this scenario
```

Comment:
Provide two examples here to help users to understand?

Reply:
Sure, added.